apps/finance/app/src/components/useFilteredTransfers.js

```diff
@@ -32,10 +32,12 @@ function useFilteredTransfers({ transactions, tokens }) {
     setSelectedDateRange(range)
   }, [])
   const handleTokenChange = useCallback(index => {
-    setSelectedToken(index || UNSELECTED_TOKEN_FILTER)
+    const tokenIndex = index === 0 ? -1 : index
```

Comment:
```suggestion
    const tokenIndex = index === 0 ? UNSELECTED_TOKEN_FILTER : index
```

Maybe this is more clear and uses the constant.